ring/ring_test.go

```diff
@@ -174,7 +172,7 @@ func TestDoBatchZeroInstances(t *testing.T) {
 }
 
 func TestDoBatch_QuorumError(t *testing.T) {
-	// we should run several write request to make sure we dont have any race condition on the batchTracker.record code
+	// we should run several write request to make sure we don't have any race condition on the batchTracker code
 	const numberOfOperations = 10000
 	instanceReturnErrors := [3]error{nil, nil, nil}
```

Comment:
this is now strangely used. We can have a constant with the number of ingesters and use that to populate the ring. I think it will be more clear if `runDoBatch` takes an array of errors instead of the tests having to share `instanceReturnErrors`